index.html

```diff
@@ -447,10 +447,6 @@ <h2 data-text="menu.utils">Utilities</h2>
           <div class="open-submenu" style="margin: auto 0 auto auto;"></div>
         </div>
         <div class="menu-hidden" data-type="treasure">
-          <div class="collection-value">
-            <a data-text="menu.hide_all" href="javascript:void(0);" onclick="Treasures.showHideAll(true)">Hide all</a>
-            <a data-text="menu.show_all" href="javascript:void(0);" onclick="Treasures.showHideAll(false)">Show all</a>
-          </div>
```

Comment:
Would really prefer these to stay available if possible.

Reply:
That is now covered by clicking on the Treasure category. They switch all on or off.

With this branch as-is, there is a cosmetic problem with enabling individual treasures after switching all off with the category button. (They stay darker in the menu until you enable all with the category button.) I’m on that.

The code is otherwise click-tested for each change (as I usually do before a pull request). To avoid future conflicts I was thinking that this minor issue can be fixed later and I do the merge request now.